test/onnx/test_fx_op_consistency.py

```diff
@@ -1559,6 +1555,18 @@ def skip_torchlib_forward_compatibility(
             "Reshape", "empty tensor"
         ),
     ),
+    skip(
+        "nn.functional.interpolate",
+        variant_name="linear",
+        matcher=lambda sample: sample.input.dtype in (torch.float16, torch.float32),
+        reason="Mismatched elements with high difference",
+    ),
+    skip(
```

Comment:
this tests of 4 all failed. however when xfail is used it failed with "Failed: Test unexpectedly passed"